RecoMET/METFilters/python/EcalBadCalibFilter_cfi.py

```diff
@@ -0,0 +1,36 @@
+import FWCore.ParameterSet.Config as cms
+ 
+ecalBadCalibSep2017ListFilter = cms.EDFilter(
+  "ecalBadCalibSep2017ListFilter",
```

Comment:
cfi file should better match the name of the module instance defined in it.
ecalBadCalibSep2017ListFilter_cfi.py
``` ecalBadCalibSep2017ListFilter = cms.EDFilter("EcalBadCalibFilter" ```